code/modules/vehicles/sealed/mecha/equipment/weapons/ballistic/automatic.dm

```diff
@@ -11,6 +11,34 @@
 	projectile_energy_cost = 20
 	fire_cooldown = 2
 
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/lmg_heavy
+	name = "\improper AC 10"
+	desc = "The original in anti-mech firepower, the standard Hephaestus Autocannon MK10 design fires AP slugs in order to damage other heavy armor suits. This does mean its rate between bursts is longer than most."
+	icon_state = "mecha_uac2"
+	equip_cooldown = 30
```

Comment:
i'd recommend doing `2 SECONDS` instead (`SECONDS` is a macro translating to `* 10` to convert to byond time in deciseconds)

it's fine to have 16% more dps than the solaris given it's non-hitscan and has less AP